### PR TITLE
Add special report alt colours

### PR DESCRIPTION
**Pull request by joecowton1**
## What is the purpose of this change?

Adds new palette colours for `specialReportAlt`.

https://www.figma.com/file/b2qv2OMLoNCYnP01ipfrP7/%E2%97%90-Colour?node-id=2840%3A4460

To avoid adding these new colours to existing colour objects and complicating the impact of this change i've created a new `special` colour array.

## What does this change?

- move all special related colours to `special` colour array
- add new special report alt palette

## Screenshots

**Before**
Screenshot 2022-09-15 at 10 23 40

**After**
Screenshot 2022-09-15 at 10 23 49

### Accessibility

- [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
- [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
- [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
- [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### Cross browser and device testing

- [ ] Tested with touch screen device

### Responsiveness

- [ ] Tested at all breakpoints
- [ ] Tested with with long text
- [ ] Stretched to fill a wide container

### Documentation

- [ ] Full API surface area is documented in the README
- [ ] Examples in Storybook

### CODE REVIEW

File: packages/@guardian/source-foundations/src/colour/palette.tsx [hunk 1]
```diff
@@ -90,6 +92,11 @@ const colors = {
 		'#ABC2C9', //specialReport-500
 		'#E4E5E8', //specialReport-700
 		'#EFF1F2', //specialReport-800
+		'#2B2B2A', //specialReportAlt-100
```
Comment (ob6160) [inline packages/@guardian/source-foundations/src/colour/palette.tsx:95]: from our conversation in chat I thought we were putting these in the `reds` array - or I miss a final decision on leaving them in here?

Comment (joecowton1) [reply packages/@guardian/source-foundations/src/colour/palette.tsx:95]: Yeah you missed that bit

**Review by tjsilver — APPROVED** [review]
A sensible arrangement!

File: packages/@guardian/source-foundations/src/colour/palette.tsx (hunk 1 above)
Comment (ob6160) [reply packages/@guardian/source-foundations/src/colour/palette.tsx:95]: sounds good 👍 - no strong opinion about where they live

**Review by ob6160 — APPROVED** [review]
⭐

**Pull request merged**